test/jdk/javax/swing/JMenuBar/TestMenuMnemonicLinuxAndMac.java

```diff
@@ -117,5 +117,7 @@ private static void createAndShowUI() {
         frame.setLocationRelativeTo(null);
         frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         frame.setVisible(true);
+        frame.toFront();
```

Comment:
` frame.toFront();` is required ? Did you try only with `frame.requestFocus();` ?